README.md

```diff
@@ -137,6 +137,28 @@ message = client.messages.create(to="+12316851234", from_="+15555555555",
                                  body="Hello there!")
 ```
 
+### Enable Debug Logging
+
+Log the API request and response data to the console:
+
+```python
+import logging
+
+client = Client(account, token)
+logging.basicConfig()
+client.http_client.logger.setLevel(logging.INFO)
```

Comment:
What's your reasoning because `logging.INFO` over `logging.DEBUG`?

Reply:
We are [currently logging](https://github.com/twilio/twilio-python/blob/48a6dfeae31419e5d9b4b207ea6d4735f318e9e6/twilio/http/http_client.py#L75) the information required (request and response data) at the `logging.INFO` level. 

That said, the `logging.DEBUG` level [includes](https://docs.python.org/3/library/logging.html#logging-levels) what's at the `logging.INFO` info, so changing it to `logging.DEBUG` may not be breaking. Currently, we are not logging anything at the `logging.DEBUG` level.